gnucash/import-export/import-main-matcher.c

```diff
@@ -977,7 +977,6 @@ gnc_gen_trans_onButtonPressed_cb (GtkTreeView *treeview,
                 selected = gtk_tree_selection_get_selected_rows (selection, &model);
                 if (gtk_tree_path_get_depth( (GtkTreePath *) selected->data) == 1)
                 {
-                    get_action_for_path (selected->data, model);
                     if (get_action_for_path (selected->data, model) == GNCImport_ADD)
```

Comment:
With the intervening `get_action_for_path` removed you can combine the if statements with `&&`.

Reply:
> With the intervening `get_action_for_path` removed you can combine the if statements with `&&`.

That's going to be a long line... Is there a suggested max line length? If it needs to be split, how would you like the line to be split?
I could not see anything about that in https://wiki.gnucash.org/wiki/CodingStandard. I do see some very long lines in Account.cpp as well as lots of different styles of line splitting.

Reply:
One should try to keep lines to less than 80 characters, but if there's no good place to break a line and you don't want to create a local variable then it's OK to go over.
But in this case there is a convenient break, right after the `&&`:
```
if (gtk_tree_path_get_depth( (GtkTreePath *) selected->data) == 1 &&
    get_action_for_path (selected->data, model) == GNCImport_ADD))
```

Reply:
Fixed and force pushed